docs-site/src/components/Search/EmptySearch.tsx

```
@@ -0,0 +1,137 @@
+          </ul>
+        </div>
+      )}
+
+      {/* Quick Links */}
+      <div className={styles.quickLinks}>
+        <span className={styles.quickLinksTitle}>Quick Links</span>
+        <div className={styles.quickLinksList}>
+          {QUICK_LINKS.map((link) => (
+            <Link
+              key={link.path}
+              to={useBaseUrl(link.path)}
```

Biome (2.3.13)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)